tools/Github/CreatePR.ps1

```diff
@@ -64,4 +66,17 @@ $Description
 * **SHOULD NOT** adjust version of module manually in pull request
 "@
 $RequestBody = @{"title" = $Title; "body" = $PrBody; "head" = $HeadBranch; "base" = $BaseBranch }
-Invoke-WebRequest -Uri https://api.github.com/repos/Azure/azure-powershell/pulls -method POST -Headers $Headers -Body ($RequestBody | ConvertTo-Json)
\ No newline at end of file
+
+$Uri = "https://api.github.com/repos/Azure/azure-powershell/pulls"
+$PullRequests = Invoke-WebRequest -Uri $Uri -Headers $Headers | ConvertFrom-Json
+$ExistingPr = $PullRequests | Where-Object { $_.title -eq $Title }
```

Comment:
I think no need to check the title, you just need to check whether there is a PR whose host is $Host and $base is $Base